templates/kubernetes/terraform/modules/kubernetes/user_auth.tf

```diff
@@ -96,6 +96,12 @@ resource "helm_release" "kratos" {
   #   # value = "${local.db_type}://${kubernetes_service.app_db.metadata[0].name}.${kubernetes_service.app_db.metadata[0].namespace}"
   # }
 
+  # This domain or address must be verified by the mail provider (Sendgrid, SES, etc.)
+  set {
+    name  = "kratos.config.courier.smtp.from_address"
+    value = "noreply@${var.domain_name}"
```

Comment:
Just realized how hidden the `mail.` prefix is,
hmm maybe we need to have an output from the `sendgrid module`, or maybe remove the prefix all together from sendgrid side https://github.com/commitdev/terraform-aws-zero/blob/d0f855816a7f135ea02425982b6ca2b1d6a1f622/modules/sendgrid/main.tf#L34

Reply:
Yeah I don't think that prefix is useful, or maybe we leave it for flexibility but default it to `""`

Reply:
https://github.com/commitdev/terraform-aws-zero/pull/28

Reply:
released v0.1.13